spec/include/PowerOptimize.vspec

```diff
@@ -0,0 +1,109 @@
+# Intermediate numbers (2-9) indicates intermediate states, where a lower number indicates a more aggressive power optimization policy.
+# 
+# This allows for subsystems to implement their own power saving logic.
+# It is totally up to the subsystem to know "what to do" for different power optimize levels.
+# It is also assumed that the subsystem knows what is required for legal and safety reasons and does not turn off
+# essential functionality regardless of power optimization level.
+# It is up to the OEM to configure subsystems to achieve the wanted behavior.
+# It is expected that power optimizations that are activated already at a high level have limited impact on the driver experience,
+# but power optimizations on lower levels will have significant impact on driver experience.
+#
+# A hypothetical example for Display/HMI brightness could be as follows
+#
+# There is a signal Vehicle.Cabin.Infotainment.HMI.Brightness that for example the driver could use to change HMI Brightness.
+# The signal has a range 0-100 percent, where 0% means the lowest value allowed from legal/safety-reasons as well as usability reasons.
+# To allow for energy optimization an OEM might want to change the actual brightness based on power optimization level for cabin.
+#
+# A possible schema could be as follows
+#
+
+#  REAL_MIN = 30 /* Lowest allowed value for safety/legal/convenience-reasons */
+#  REAL_MAX = 90 /* Maximum allowed value for safety/legal/convenience/technical-reasons */
+#  POWERSAVE_MAX_RELATIVE = 45 /* Maximum relative brightness during power optimize */
+#  POWERSAVE_LIMIT = 5 /* Reduce brightness if level is below this value */
+#
+#  if (Vehicle.Cabin.PowerOptimize. < POWERSAVE_LIMIT):
+#    old_value = Vehicle.Cabin.Infotainment.HMI.Brightness;
+#    if (Vehicle.Cabin.Infotainment.HMI.Brightness > POWERSAVE_MAX_RELATIVE):
+#      Vehicle.Cabin.Infotainment.HMI.Brightness = POWERSAVE_MAX_RELATIVE;
+#  else
+#    // Revert to old value
+#    Vehicle.Cabin.Infotainment.HMI.Brightness = old_value;
+#
+#
+# Note that no additional VSS signals is added to represent wanted brightness at power optimization.
+# It is possible that a such signal may be needed in individual deployments, but if so it can be handled by a vehicle specific overlay.
+#
+# Customization
+#
+# In addition to power optimize signals added in standard VSS catalog a project/OEM may customize the model by overlay by e.g.
+#
+# 1. Adding additional power optimize level signals
+#
+# For instance by adding `#include include/PowerOptimize.vspec` in additional files
+#
+# 2. Adding signals for supporting data
+#
+# In specific deployments there might be a need for private signals with additional data for the power optimization strategy used.
+# This could for instance be signals to specify maximum speed for a specific power optimization levels.
+
+##
+# Power Optimization
+##
+
+  
+PowerOptimizeLevel:
+  datatype: uint8
+  type: actuator
+  min: 1
+  max: 10
```

Comment:
Meeting notes: Info from EV Power optimize group, shall be refactored, 0 =no power optimize, 10= maximum. If redefined no problem to be merged.

Way forward:

- Erik to update PR
- Mobis/Umang to review after change
- If OK, Erik can merge

Reply:
PR updated, @UmangSharmaMobis please review